Gemfile

```diff
@@ -5,13 +5,12 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
 ruby '2.3.1'
```

Comment:
It's not _totally_ necessary, but when you're doing a refactoring of this scale, it's probably worth bringing the ruby version up to our current standard (i.e. 2.3.3) whilst you're at it.